Version 1.3.1 only shows one playlist

**Describe the bug**
Version 1.3.1 shows only one playlist.

**To Reproduce**
1. Open ncspot
2. Go to the playlist section
3. Only one playlist shows up for me

**Expected behavior**
All playlists of my account should be shown.

**Screenshots**

<img width="1488" height="1366" alt="Image" src="https://github.com/user-attachments/assets/c1ad53db-d8bd-451f-9668-5530734c4d55" />

**System (please complete the following information):**
 - OS: Linux (Arch)
 - Terminal: Konsole
 - Version: 1.3.1
 - Installed from: AUR (ncspot-bin, ncspot-ncurses), so the binary release and a self-built one

**Additional context**
I already tried deleting `~/.cache/ncspot` entirely and logging in again, but this didn't fix the issue.

Thank you very much for releasing a new version using librespot 0.7.0 and thus fixing the bug so quickly! However, without the playlists not properly working, the restored functionality is somehow still limited. Maybe this is something in librespot rather than in ncspot, but I am not able to assess this.

I managed to reproduce, I can only see two playlists. 

```
[2025-08-26][08:43:56] [ncspot::ui::pagination] [DEBUG] fetching next page at offset 200
[2025-08-26][08:43:56] [ncspot::spotify_api] [DEBUG] fetching playlist 3ZcAJLSa7T9QpqCgIG2MAV tracks, offset: 200
[2025-08-26][08:43:56] [rspotify_http::ureq] [INFO] Making request Request(GET https://api.spotify.com/v1/playlists/3ZcAJLSa7T9QpqCgIG2MAV/tracks, [authorization: Bearer BQAlv0pQlp4WU-kLl2z_2ALffoaSB35sehx2TxOng2hGp4K_5kMHo2OUDmwMlAoMqpmUdLf88991JgsDhe__JYSOvkrutcyJY6lyDJQYQn-musuYlVzS4ueWPFri0cM-PPFKkNZ5feQnP2-wT8u34zUOUbcqvlBb1GP6_VEBTmjFSdV7ThhoH3KIAaX2ulN54pKuCfZseygnFomLdDMRp4tz4abYQ97Pg42Xa431NKpWaa98SgrWq46J--pA74kvaKzRDqQh_QPDrSmxaVnvBmsPPcjW3J0lF0MXFfRTcuqqSPIoiWn_TdhN_Tw2v8sERPS6cfLUEaXsdZtuexDXBw2SOy1qz7HCCKdo31XjV6gpB_mXJUBUfebuBaLzrKUykWq48V1lgMBGrb8])
[2025-08-26][08:43:56] [ureq::pool] [DEBUG] pulling stream from pool: https|api.spotify.com|443 -> Stream(SslStream { stream: TcpStream { addr: [2a0c:5a84:b206:4500:741c:9147:cf1b:6882]:41074, peer: [2600:1901:1:7c5::]:443, fd: 19 }, ssl: Ssl { state: "SSL negotiation finished successfully", verify_result: X509VerifyResult { code: 0, error: "ok" } } })
[2025-08-26][08:43:56] [ureq::unit] [DEBUG] sending request (reused connection) GET https://api.spotify.com/v1/playlists/3ZcAJLSa7T9QpqCgIG2MAV/tracks?offset=200&market=from_token&limit=100
[2025-08-26][08:43:56] [ureq::unit] [DEBUG] writing prelude: GET /v1/playlists/3ZcAJLSa7T9QpqCgIG2MAV/tracks?offset=200&market=from_token&limit=100 HTTP/1.1
Host: api.spotify.com
User-Agent: ureq/2.12.1
Accept: */*
authorization: ***
[2025-08-26][08:43:57] [ureq::response] [DEBUG] Streaming body until content-length: 235505
[2025-08-26][08:43:57] [ureq::unit] [DEBUG] response 200 to GET https://api.spotify.com/v1/playlists/3ZcAJLSa7T9QpqCgIG2MAV/tracks?offset=200&market=from_token&limit=100
[2025-08-26][08:43:57] [ureq::pool] [DEBUG] adding stream to pool: https|api.spotify.com|443 -> Stream(SslStream { stream: TcpStream { addr: [2a0c:5a84:b206:4500:741c:9147:cf1b:6882]:41074, peer: [2600:1901:1:7c5::]:443, fd: 19 }, ssl: Ssl { state: "SSL negotiation finished successfully", verify_result: X509VerifyResult { code: 0, error: "ok" } } })
[2025-08-26][08:43:57] [ncspot::spotify_api] [ERROR] unhandled api error: json parse error: data did not match any variant of untagged enum PlayableItem at line 1 column 33128
```

```
[2025-08-26][08:43:57] [ncspot::spotify_api] [ERROR] unhandled api error: json parse error: data did not match any variant of untagged enum PlayableItem at line 1 column 33128
```
This seems to suggest that `rspotify` was unable to deserialize one of the items in the playlist. Anything special in there? Podcast episode? Local file?

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Fixed
+
+- Playlist retrieval crashing when list contains podcast episodes
+
 ## [1.3.1]
 
 ### Fixed

--- a/src/model/playable.rs
+++ b/src/model/playable.rs
@@ -144,6 +144,7 @@ impl From<&PlayableItem> for Playable {
         match item {
             PlayableItem::Episode(episode) => Self::Episode(episode.into()),
             PlayableItem::Track(track) => Self::Track(track.into()),
+            PlayableItem::Unknown(value) => panic!("Unknown playable item {value:?}"),
         }
     }
 }

--- a/src/spotify_api.rs
+++ b/src/spotify_api.rs
@@ -431,6 +431,16 @@ impl WebApi {
                         items: page
                             .items
                             .iter()
+                            .filter(|pt| {
+                                if let Some(t) = pt.track.as_ref()
+                                    && !t.is_unknown()
+                                {
+                                    true
+                                } else {
+                                    error!("Could not process item {pt:?}, ignoring");
+                                    false
+                                }
+                            })
                             .enumerate()
                             .flat_map(|(index, pt)| {
                                 pt.track.as_ref().map(|t| {

